bundles/org.openhab.binding.pihole/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.pihole</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: PiHole Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
```

Comment:
Why is this dependency required ?

Reply:
To run unit tests and have logging

Reply:
Take a look at how it's done in other [bindings](https://github.com/search?q=repo%3Aopenhab%2Fopenhab-addons%20qos.logback&type=code) . Tests should not be in the binding pom apparently.

Reply:
You are talking about integration tests not unit https://www.openhab.org/docs/developer/tests.html#unit-tests

Reply:
Didn't check details, but might help:
https://github.com/openhab/openhab-addons/blob/a9f7e0fef74b78efcc11ee09918e18432d4ccfe3/bundles/org.openhab.binding.hdpowerview/pom.xml#L23-L30

https://github.com/openhab/openhab-addons/blob/a9f7e0fef74b78efcc11ee09918e18432d4ccfe3/bundles/org.openhab.binding.hdpowerview/src/test/java/org/openhab/binding/hdpowerview/internal/AutomationChannelBuilderTest.java#L37-L38

Reply:
You both see that this dep is in test scope, right?

```xml
 <dependencies>
    <dependency>
      <groupId>ch.qos.logback</groupId>
      <artifactId>logback-classic</artifactId>
      <version>1.2.3</version>
      <scope>test</scope> <!-- here -->
    </dependency>
```